2-mandatory/4-stocks.js

```diff
@@ -47,8 +53,12 @@ function getAveragePrices(closingPricesForAllStocks) {
                 (Apple's price on the 5th day) - (Apple's price on the 1st day) = 172.99 - 179.19 = -6.2
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
-function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+
+function getPriceChanges(closingPricesForAllStocks)
+{
+    const PriceChange = (closingPricesForAllStocks.map((array) => array[array.length - 1] - array[0])).map((element) => parseFloat(element.toFixed(2)));
+
+    return PriceChange;
```

Comment:
```suggestion
function getPriceChanges(closingPricesForAllStocks)
{
    const priceChange = closingPricesForAllStocks.map((array) => array[array.length - 1] - array[0])
    const result = priceChange.map((element) => parseFloat(element.toFixed(2));
    return result;
```
It's better to use to variables to make code more readable